examples/next/components/filters.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {
+  RefinementList,
+  connectStats,
+} from 'react-instantsearch-dom';
+
+function Filters() {
+  return (
+    <RefinementList attribute="categories" />
```

Comment:
Hey @callmephilip thanks for the contribution.
It's not clear to me why wrapping Filters with connectStats needed. Can you explain to us your usecase?

Nesting connectors (in this case `connecStats` & `connectRefinementList`) is in general not advisable because they can cause too many renderings.